### ISSUE TITLE
Batch normalisation alternative

### ISSUE THREAD

**Issue by KirillDogadin-std** [KirillDogadin-std/Lab-Semantic#6]
https://www.tensorflow.org/api_docs/python/tf/compat/v1/layers/batch_normalization

The batch normalistation in `tf` has an `axis parameter`

For [torch nn](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html) and for [functional](https://pytorch.org/docs/stable/nn.functional.html#batch-norm) i dont find the axis. Do I miss something? Googling didnt provide a bold result, thus here's the question/request.

What is the better way to specify axis? Did you have similar issues in the past with some of the `torch` units? 

Also, I assume that axis splecification is crutial, thus comes the question.

**Comment by 4rash-uni** [comment]
Reshape it with view to NxCxHxW do batch norm and again reshape it back to default size, keep in mind that the normalisation perform on C in tensorflow when axis=1.

**Comment by KirillDogadin-std** [comment]
Thanks!
